docs/edit/features.md

```diff
@@ -15,6 +15,21 @@ class Test_AwesomeFeature:
 
 Several key points:
 
+* Each new feature should be defined in [_features.py](/utils/_features.py). In most cases this consists of copying one of the already added features, changing the name, bumping the number in the url, and bumping the feature number. In this case we'd add
```

Comment:
Discussing on slack, but adding here for tracking purposes --
Your changes to edit/README and edit/features will conflict with my PR[.](https://github.com/DataDog/system-tests/pull/3456) . Why don't I just fold your changes into mine?